src/components/Navbar/Navbar.js

```diff
@@ -48,24 +53,30 @@ const Navbar = () => {
         <div className="col-lg-2">
           <GoThreeBars />
         </div>
-        <div className="col-lg-8">Wordle</div>
-        <div className="col-lg-2 navrside">
+        <div className="col-lg-3">Wordle</div>
+        <div className="col-lg-7 navrside">
```

Comment:
Why was col-lg-8 and col-lg-2 changed to col-lg-3 and col-lg-7?

Reply:
Was attempting to use grid system in navbar. However, animation icons did not align correctly so I ended up not using animation icons and instead focused on react-query.